rivertile/main.zig

```diff
@@ -298,6 +298,7 @@ const Output = struct {
                     .bottom => layout.commit("rivertile - bottom", ev.serial),
                 }
             },
+            else => {},
```

Comment:
I think it's a bit more readable to have this explicit.
```suggestion
            .user_command_tags => {},
```